### ISSUE TITLE
Blackwidow V4 

### ISSUE THREAD

**Issue by GoofyDLoofy** [1kc/razer-macos#853]
Will there ever be an update for the normal BlackWidow V4? Not the pro or the 75%.

**Comment by stickoking** [comment]
I don't think this repo is being maintained anymore I have made a fork can you post your issue there? I will try to add the v4 I have just added the 75%. https://github.com/stickoking/razer-macos

**Comment by GoofyDLoofy** [comment]
How do you make a fork? I am new here.

**Comment by stickoking** [comment]
By clicking the fork button on the repo you want to fork

**Comment by GoofyDLoofy** [comment]
like that?
